scenes/cslib_array.txt

```diff
@@ -247,6 +247,180 @@
 *return
 
 
+*comment TO_STRING
```

Comment:
I suggest we use another name. Even in serious programming languages "to\_string" is not used to mean joining elements in an array. At least I haven't seen it. Maybe "join"?

Reply:
I agree! "Join" would have been my first choice, but I'm trying to think as if I'm not familiar with programming. What is the most intuitive name we can give it?

I'm not married to "to string" but maybe join isn't the best? Maybe one to discuss on discord.